tests/phpunit/admin/class-wc-rest-stripe-settings-controller-test.php

```
@@ -136,6 +136,55 @@ public function test_update_stripe_payment_method_configurations_settings() {
+						'message' => 'Invalid configuration.',
+					],
+				]
+			);
+
+		$request = new WP_REST_Request( 'POST', self::SETTINGS_ROUTE );
+		$request->set_param( 'enabled_payment_method_ids', [ 'amazon_pay', 'card' ] );
+		$request->set_param( 'is_upe_enabled', true );
+
+		$response = $this->controller->update_settings( $request );
+		$this->assertEquals( 500, $response->get_status() );
+	}
```

PHPMD (2.15.0)

[error] The method test_update_settings_returns_500_when_pmc_update_fails is not named in camelCase. (undefined)

(CamelCaseMethodName)